specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/streamingservice.json

```diff
@@ -1549,21 +1768,21 @@
       "properties": {
         "description": {
           "type": "string",
-          "description": "The StreamingEndpoint description."
+          "description": "The streaming endpoint description."
         },
         "scaleUnits": {
           "type": "integer",
           "format": "int32",
-          "description": "The number of scale units.  Use the Scale operation to adjust this value."
+          "description": "The number of scale units. Use the Scale operation to adjust this value."
         },
         "availabilitySetName": {
           "type": "string",
-          "description": "The name of the AvailabilitySet used with this StreamingEndpoint for high availability streaming.  This value can only be set at creation time."
+          "description": "The name of the AvailabilitySet used with this streaming endpoint for high availability streaming. This value can only be set at creation time. This feature is not available everywhere, please contact Azure Support for more information."
```

Comment:
Is this feature still in use and available?

Reply:
Pending email discussion.

Reply:
ning has updated the description to add deprecated